src/gameboy/src/gameboy.cpp

```diff
@@ -14,7 +26,7 @@ Gameboy::Gameboy(std::string rom_path, bool debug_on) {
 	memory->set_cpu(cpu.get());
 	memory->set_gpu(gpu.get());
 
-	Log::info("Gameboy start successful!");
+
```

Comment:
I think you can leave it in here, it marks that all modules have completed initialization